lua/autorun/ttt2_force_download.lua

```diff
@@ -18,6 +18,8 @@ if SERVER then
 	resource.AddFile("materials/vgui/ttt/dynamic/roles/icon_traitor.vmt")
 	resource.AddFile("materials/vgui/ttt/dynamic/roles/icon_det.vmt")
 	resource.AddFile("materials/vgui/ttt/dynamic/roles/icon_no_team.vmt")
+	
+	resource.AddFile("materials/vgui/ttt/equip/reroll.png")
```

Comment:
I just added this to the downloadlist, it has nothing todo with the armor system, but I commited it to the wrong branch. I can change it though

Reply:
Hmm, I changed it to a vmt/vtf, now it is the only icon of the shop tabs that is a vmt/vtf. All the others are pngs. Therefore I think it should stay a png

Reply:
Wether or not, this should be moved out of this PR

Reply:
Since I did not change anything besides adding a line to the forcedownload list, I see no point in removing it here and adding it manually to the master.

Yes, it does not belong in here, but adding two commits for one line of code seems pointless to me